core/src/main/java/org/kohsuke/stapler/PostProcessor.java

```diff
@@ -0,0 +1,7 @@
+package org.kohsuke.stapler;
+
+/**
+ * @author <a href="mailto:nicolas.deloof@gmail.com">Nicolas De Loof</a>
+ */
+public class PostProcessor {
```

Comment:
Is this supposed to be an annotation or have other stuff?

Reply:
seems to be some draft code I committed by mistake

Reply:
@ndeloof, it seems like this draft code should get cleaned up before merging. Is that something you could do?